DOCSP-37406 Add docs release notes for w/e March 8 (#3200)
mongodb/docs-realm: docs-release-notes.md

# March 8, 2024

## Node.js SDK

- Landing Page: Remove link to outdated Electron guide.

> - Integration Guides: Remove Integration Guides section from TOC, as these Electron guides are outdated.

Reviewer: Question - I wonder if it would make sense to mention the fact that we do intend to add these guides back in the future after we've had the opportunity to update them?

```suggestion
- Integration Guides: Remove outdated Integration Guides section from TOC until we have time to update the guides.
```
